Solutions/Azure Active Directory/Analytic Rules/AuthenticationMethodsChangedforPrivilegedAccount.yaml

```diff
@@ -28,7 +28,7 @@ query: |
       | where TimeGenerated > ago(queryperiod)
       | mv-expand AssignedRoles
       | where AssignedRoles matches regex 'Admin'
```

Comment:
can we use here 'has' or 'contains' instead of regex matching?

Reply:
Based on the testing I've done, 'contains' works.

Reply:
looks good. thanks for helping us improve our content.